packages/figma-utils/.gitignore

```diff
@@ -0,0 +1 @@
+variables*.css
```

Comment:
sanity question/just to be safe: this does not affect the other package folders, right?

Reply:
Nope only the `figma-utils` package so that when we play around locally for testing these files are automatically ignored by git